src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -763,7 +762,7 @@ internal async Task SearchPackagesAndRefreshUpdateCountAsync(string searchText,
                 {
                     // it means selected tab is update itself, so just wait for searchAsyncTask to complete
                     // without making another call to loader to get all packages.
-                    _topPanel._labelUpgradeAvailable.Count = 0;
+                    _topPanel.ShowCountOnUpdatesTab(count: 0);
```

Comment:
Perhaps "UpdateCountOnUpdatesTab(int count)" would be better naming?

Reply:
Went with SetCountOnUpdatesTab... Let me know if that doesn't work.